cypress/integration/main_page_spec.js

```diff
@@ -213,7 +213,8 @@ describe('Dashboard Home Page', () => {
       // Cypress only allows one stub per url. We make multiple POST requests to the same
       // elasticsearch endpoint. The fixture here returns a combined response of all the
       // responses for one url, effectively stubbing our elasticsearch searches.
-      cy.route('POST', 'http://example.com/_search/', 'fixture:elasticsearch.json');
+      let fixtureName = 'elasticsearch.json';
+      cy.intercept('POST', 'http://example.com/_search/', (req) => req.reply({ statusCode: 200, fixture: fixtureName }));
```

Comment:
`cy.intercept` doesn't allow you to overwrite an intercepted URL (https://stackoverflow.com/questions/65044704/change-fixture-response-in-cypress-for-the-same-url-with-intercept). So this is a way to change the fixture.  It's seems inelegant, but I don't have a better solution.